src/Sharpbrake.Client/WebProxy.cs

```diff
@@ -1,25 +1,23 @@
-﻿#if NETSTANDARD1_4 || NETSTANDARD2_0
```

Comment:
Is there any explanation to this change? Did `NETSTANDARD2_0` started supporting the `WebProxy` API?

Reply:
Yes